recipe/meta.yaml

```diff
@@ -8,10 +8,17 @@ source:
   fn: Pillow-{{ version }}.tar.gz
   url: https://github.com/python-pillow/Pillow/archive/{{ version }}.tar.gz
   sha256: eb4b9c2956e38f89d588a0d0af0e27a03f1e1d2be705a828bbd12cfba0b020fa
+  patches:
+    # Apply the diff from a Pillow PR to add the `--disable-osx-tcltk-framework`
+    # flag. Link below.
+    #
+    # https://github.com/python-pillow/Pillow/pull/1883
+    #
+    - pillow_PR_1883.diff
 
 build:
   number: 2
-  script: pip install --no-deps .
+  script: python setup.py build_ext --debug --disable-osx-tcltk-framework install --single-version-externally-managed --record=record.txt
```

Comment:
I've just committed a PR in pillow (https://github.com/python-pillow/Pillow/pull/1932) that makes the `--disable-osx-tcltk-framework` flag unnecessary. We will be reverting the PR that added the framework flag before the next release.